revpi_core.c

```diff
@@ -58,6 +58,11 @@ u8 revpi_core_find_gate(struct net_device *netdev, u16 module_type)
 {
 	int i;
 
+	if (piDev_g.machine_type != REVPI_CONNECT_SE){
```

Comment:
What is this supposed to do? Should this be == instead of != ?

Reply:
good catch, it should be ==